.aegir.js

```diff
@@ -21,6 +22,39 @@ module.exports = {
       included: false
     }],
     browserNoActivityTimeout: 100 * 1000,
+    reporters: ['mocha-own', 'coverage-istanbul'],
+    coverageIstanbulReporter: {
+      reports: ['json'],
+      dir: path.join(__dirname, 'coverage'),
+      combineBrowserReports: true,
+      fixWebpackSourcePaths: true,
+    },
+    preprocessors: { 'node_modules/aegir/src/config/karma-entry.js': [ 'webpack', 'sourcemap' ] },
+    webpack: {
+      module: {
+        rules: [
+          // instrument only testing sources with Istanbul
+          {
+            test: /\.js$/,
+            use: { loader: 'istanbul-instrumenter-loader' },
+            include: path.resolve('src/')
+          }
+        ]
+      }
+    },
+    customLaunchers: {
+      ChromeDocker: {
+        base: 'ChromeHeadless',
+        // We must disable the Chrome sandbox when running Chrome inside Docker (Chrome's sandbox needs
+        // more permissions than Docker allows by default)
+        flags: ['--no-sandbox']
+      }
+    },
+    client: {
+      mocha: {
+        bail: true,
+      }
+    },
```

Comment:
Will some of this eventually move into Aegir? Seems weird that we have so much Aegir config overrides when it is basically built to service PL JS projects.

Reply:
we are adding features here that aegir doesn't support yet, but it will and this will all be removed.